Source/MQTTnet/Implementations/MqttTcpChannel.Uwp.cs

```diff
@@ -85,7 +85,7 @@ public async Task ConnectAsync(CancellationToken cancellationToken)
                     socketProtectionLevel = SocketProtectionLevel.Tls10;
                 }
 
-                await _socket.ConnectAsync(new HostName(_options.Server), _options.GetPort().ToString(), socketProtectionLevel);
```

Comment:
why did you remove the socketProtectionLevel

Reply:
Sorry that wasn't my intent, just fixed it.